protostar/utils/log_color_provider.py

```diff
@@ -36,9 +36,9 @@ def get_color(
     def colorize(self, color_name: SupportedColorName, content: str):
         return f"{self.get_color(color_name)}{content}{self.get_color('RESET')}"
 
-    # pylint: disable=no-self-use
-    def bold(self, content: str):
-        return f"{Style.BRIGHT}{content}{Style.RESET_ALL}"
+    @staticmethod
+    def bold(content: Union[str, int]):
```

Comment:
```suggestion
    def bold(content: Any):
```

so that it would match `print` or `str.format` signature